content/_policy/privacy._es.md

```diff
@@ -24,30 +24,30 @@ Login.gov se asocia con agencias que necesitan acceso seguro y privado a sus apl
-* Para comprobar la identidad se requiere información de carácter confidencial, como el número de seguridad social, su dirección, el número de teléfono asociado a su dirección y el documento de identidad estatal o el carné de conducir estadounidense. El operador de telefonía móvil (cualquier operador de telefonía móvil de la marca) puede revelar su número de móvil, nombre, dirección, correo electrónico, estado de la red, tipo de cliente, rol de cliente, tipo de facturación, identificadores del dispositivo móvil (IMSI e IMEI) y otros detalles del estado del suscriptor y del dispositivo, si están disponibles, a nuestro proveedor de servicios externo, con el único fin de verificar su identidad mientras dure la relación comercial.
+* Para comprobar la identidad se requiere información de carácter confidencial, como el número de seguridad social, su dirección, el número de teléfono asociado a su dirección y el documento de identidad estatal o el carné de conducir estadounidense. Su operador de telefonía móvil (cualquier operador de marca) puede revelar su número de teléfono, nombre, dirección, correo electrónico, estado de la red, tipo de cliente, función de cliente, tipo de facturación, identificadores del dispositivo móvil (IMSI e IMEI) y otros datos del estado del usuario y del dispositivo, si están disponibles, a nuestro proveedor de servicios externo, únicamente para verificar su identidad mientras dure la relación comercial.
 * Para mitigar el fraude, también analizamos el dispositivo utilizado para acceder a Login.gov, la identidad y el comportamiento, por ejemplo, cómo interactúa con los formularios de la página.
 
 ### Usos de rutina: ¿con quién se comparte habitualmente la información? ### {#routine-uses}
 
-* A los servicios de prueba de identidad de terceros, según sea necesario para probar la identidad de una persona para acceder a un servicio con el nivel de garantía requerido.
+* A los servicios de comprobación de identidad de terceros, según sea necesario para comprobar la identidad de un individuo para el acceso a un servicio.
 * A un experto, consultor o contratista de GSA en el desempeño de un deber federal para el cual la información es relevante.
 * A la Oficina de Publicaciones del Gobierno (GPO), cuando Login.gov necesita enviar a un usuario un formulario de confirmación de dirección o si un usuario solicita notificaciones enviadas por correo de cambios de cuenta o de intentos de revisión.
 
-Esta lista no es exhaustiva. Consulte el aviso del sistema de registros para todas las entidades y personas.
+Esta lista no es exhaustiva. Por favor, consulte el aviso del [sistema de registros GSA/TTS-1](https://www.federalregister.gov/documents/2017/08/10/2017-16852/privacy-act-of-1974-system-of-records) para todas las entidades y personas.
```

Comment:
In English, "notice" is included in the link text. Should it be here as well for consistency?

```suggestion
Esta lista no es exhaustiva. Por favor, consulte [el aviso del sistema de registros GSA/TTS-1](https://www.federalregister.gov/documents/2017/08/10/2017-16852/privacy-act-of-1974-system-of-records) para todas las entidades y personas.
```